prefect_gcp/cloud_storage.py

```diff
@@ -1058,7 +1068,9 @@ async def upload_from_folder(
         for from_path in from_folder.rglob("**/*"):
             if from_path.is_dir():
                 continue
-            bucket_path = str(Path(bucket_folder) / from_path.relative_to(from_folder))
+            bucket_path = str(
+                Path(bucket_folder or "") / from_path.relative_to(from_folder)
```

Comment:
Should the `or ""` new moved up to line 1062 so that it doesn't need to be repeated?